test/support/postgresql/import_examples.rb

```diff
@@ -78,6 +78,24 @@ def should_support_postgresql_import_functionality
       end
     end
 
+    describe "with compound foreign keys" do
```

Comment:
You'll want to wrap this in a `unless ENV["SKIP_COMPOSITE_PK"]` condition since composite_primary_keys gem doesn't support Rails 7 yet.

Reply:
Oh, thank you! I was trying to understand what's wrong with Rails 7!